app/controllers/api/v2/base_controller.rb

```diff
@@ -168,6 +166,9 @@ def render_error(error, options = { })
         render options.merge(:template => "api/v2/errors/#{error}",
           :layout => 'api/v2/layouts/error_layout')
       end
+
+      def self.hide_taxonomy_options
+      end
```

Comment:
Waiting for revert to a previous state? :)

Reply:
Oof, no idea how that happened but I must've force pushed some code that I was testing out. Thank you for pointing this out, I will fix it.

Reply:
Okay, now it should be in the previous state. Works as expected.